odoo/fields.py

```diff
@@ -2915,7 +2953,7 @@ def convert_to_write(self, value, record):
         if isinstance(value, BaseModel) and value._name == self.comodel_name:
             # make result with new and existing records
             inv_names = {field.name for field in record._field_inverses[self]}
-            result = [(6, 0, [])]
+            result = [(X2ManyCmd.SET, 0, [])]
```

Comment:
We should use the command functions here:
```suggestion
            ids = []
            result = [Command.set(ids)]
```
and use `ids.append()` below

Reply:
Bruh... I don't like that kind of magic where a mutable structure is sent empty to a function and later is feeded with data.

Reply:
why not `result = [Command.set([])]`?